.env.example

```diff
@@ -0,0 +1,3 @@
+## Example Config Here
+## Be careful to not use real keys
+TOKEN="123456789:ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
So I'm confused, do I put my token in a `.env` or not? the comment here says not to, but the description says to use a `.env` instead of a `config.json`. Is this comment saying not to put it in the example one?